app/styles/_design-system.scss

```diff
@@ -10,6 +10,7 @@
 @import 'node_modules/design-system/dist/scss/wds-components/lists';
 @import 'node_modules/design-system/dist/scss/wds-components/global-footer/common';
 @import 'node_modules/design-system/dist/scss/wds-components/global-footer/breakpoint-l-down';
+@import 'node_modules/design-system/dist/scss/wds-components/global-footer-wikia-org';
```

Comment:
I wonder if we could load these styles conditionally only for wikia.org communities?